hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/test/java/org/apache/hadoop/mapreduce/lib/output/committer/manifest/TestRenameStageFailure.java

```diff
@@ -121,6 +121,7 @@ protected boolean requireRenameResilience() {
 
   @Test
   public void testResilienceAsExpected() throws Throwable {
+    Assume.assumeTrue(etagsPreserved);
```

Comment:
What if its HNS account and etags are not preserved due to code-issue, this test will be ignored and code shall be merged in trunk. Is it possible to know if its non-HNS account and then ignore on basis of that?

Reply:
The final changes made return different values of requireRenameResilience based on whether it is a nonHNS or HNS account. Thus, the problem of going by the value of etagsPreserved is solved.